### PR TITLE
Add talend integration guide

### PR DESCRIPTION
**Pull request by petedejoy**
Converting the doc from notion to a markdown file compatible with our guides site. 

@vparekh94 @kentdanas can I get a review here to make sure all looks ok?

### CODE REVIEW

**Review by kentdanas — COMMENTED** [review]
Added a couple of small comments but other than that looks great to me!

File: guides/airflow-talend-integration.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+# environment namespace when deployed to Astronomer.
+if namespace =='default':
+    config_file = '/usr/local/airflow/include/.kube/config'
+    in_cluster=False
+else:
+    in_cluster=True
+    config_file=None
+
+# Define recipient emails for successful completion notification
+email_to = ["kenten@astronomer.io"]
```
[the remark this answers is not in the document]
Comment (petedejoy) [reply guides/airflow-talend-integration.md:119]: Of course- good catch.

File: guides/airflow-talend-integration.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+|Pros        |- Containerizing jobs brings the benefits of containerization including efficiency, flexibility, and scaling - Easily allows for downstream dependencies  - Logs from jobs are shown in Airflow UI|- Very easy and accessible. Little setup and knowledge of other tools is required                                                   |
+|Cons        |- Must have Talend Studio to containerize jobs - More requirements and complexity to setup                                                                                                        |- Not well suited for triggering jobs that have downstream dependencies - Logs from Talend job are not automatically sent to Airflow|
+|Requirements|- Talend Studio license - Docker registry (can use Dockerhub if public is okay) - Kubernetes                                                                                                      |- Talend cloud license that allows API access                                                                                       |
```
[the remark this answers is not in the document]
Comment (petedejoy) [reply guides/airflow-talend-integration.md:36]: Hmm, will push up a fix shortly.

**Review by kentdanas — APPROVED** [review]

File: guides/airflow-talend-integration.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+                in_cluster=in_cluster, # if set to true, will look in the cluster, if false, looks for file
+                cluster_context='docker-desktop', # is ignored when in_cluster is set to True
+                config_file=config_file,
+                is_delete_operator_pod=True,
+                get_logs=True
+                )
+
+    talend2 = KubernetesPodOperator(
+                namespace=namespace,
+                image="davidkoenitzer/talendjob:random",
```
Comment (vparekh94) [inline guides/airflow-talend-integration.md:140]: lets change this image name to be less specific

File: guides/airflow-talend-integration.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+![Airflow Talend Task Log 2](https://assets2.astronomer.io/main/guides/talend/airflow_talend_3.png)
+
+You can also view task statuses as you would normally using the Airflow UI. If your Talend job runs successfully, the Airflow task that executed it will have a 'success' status. If the Talend job fails, the Airflow task will be marked 'failed' (as shown on the second task in the screenshot below), and any error messages will be printed in the Airflow task logs.
+
+![Airflow Task Statuss](https://assets2.astronomer.io/main/guides/talend/airflow_talend_4.png)
+
+Pretty straight forward! Once your Talend jobs are containerized, they can be orchestrated using Airflow in any sort of DAG needed for your use case, by themselves, with other Talend jobs, with any other Airflow tasks, etc. 
+
+## Executing Talend Jobs Using the Cloud API
```
Comment (vparekh94) [inline guides/airflow-talend-integration.md:189]: Small thing, but can we move the CloudAPI piece to be first? I think more users would want to do things this way

**Review by vparekh94 — CHANGES_REQUESTED** [review]

File: guides/airflow-talend-integration.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+`<Username>` Your DockerHub username 
+
+`<Password>` Your DockerHub Password 
+
+When you select 'Finish' the job will be converted into Docker image and pushed to the indicated registry. In this example the job was pushed to [https://hub.docker.com/repository/docker/davidkoenitzer/talendjob](https://hub.docker.com/repository/docker/davidkoenitzer/talendjob) 
+
+Talend can also publish jobs to Amazon ECR, Azure ACR, and Google GCR. Use this guide from Talend for connection parameter specifics [https://www.talend.com/blog/2019/03/12/how-to-deploy-talend-jobs-as-docker-images-to-amazon-azure-and-google-cloud-registries/](https://www.talend.com/blog/2019/03/12/how-to-deploy-talend-jobs-as-docker-images-to-amazon-azure-and-google-cloud-registries/)
+
+![Executing%20Talend%20Jobs%20with%20Airflow%202a2beb0aae414e13b1fcf6db0c53e173/Screen_Shot_2020-10-28_at_3.37.35_PM.png](Executing%20Talend%20Jobs%20with%20Airflow%202a2beb0aae414e13b1fcf6db0c53e173/Screen_Shot_2020-10-28_at_3.37.35_PM.png)
```
Comment (kentdanas) [inline guides/airflow-talend-integration.md:156]: One more comment - the link for this screenshot is showing up as broken for me, but I wasn't sure how to fix it.

File: guides/airflow-talend-integration.md [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+---
+
+## Why use Talend and Airflow
+
+[Talend](https://www.talend.com/) is a popular tool for data integration and data management that can be easily used along with Airflow and Astronomer to have the best of multiple worlds for data management. 
+
+There are a couple of benefits to using Airflow and Talend together:
+
+- Using Airflow for orchestration allows for easily running multiple jobs with dependencies, parallelizing jobs, monitoring run status and failures, and more
+- Combining Talend and Airflow allows you to use both tools for what they're good for. If Talend works particularly well for one use case and python for another, you can do both and still have a one-stop-shop for orchestration, monitoring, logs, etc. with Airflow and Astronomer
```
Comment (mik-laj) [inline guides/airflow-talend-integration.md:17]: I think Python should be capitalized.

```diff
-- Combining Talend and Airflow allows you to use both tools for what they're good for. If Talend works particularly well for one use case and python for another, you can do both and still have a one-stop-shop for orchestration, monitoring, logs, etc. with Airflow and Astronomer
+- Combining Talend and Airflow allows you to use both tools for what they're good for. If Talend works particularly well for one use case and Python for another, you can do both and still have a one-stop-shop for orchestration, monitoring, logs, etc. with Airflow and Astronomer
```

Comment (petedejoy) [reply guides/airflow-talend-integration.md:17]: Thanks Kamil- working on getting that change in here. Appreciate you helping out! https://github.com/astronomer/airflow-guides/pull/151

**Pull request merged**